content/pages/publishAlgo.json

```diff
@@ -0,0 +1,69 @@
+{
+  "title": "Publish",
+  "description": "Highlight the important features of your algorith to make it more discoverable and catch the interest of data consumers.",
+  "warning": "Given the beta status, publishing on Ropsten or Rinkeby first is strongly recommended. Please familiarize yourself with [the market](https://oceanprotocol.com/technology/marketplaces), [the risks](https://blog.oceanprotocol.com/on-staking-on-data-in-ocean-market-3d8e09eb0a13), and the [Terms of Use](/terms).",
+  "form": {
+    "title": "Publish",
+    "data": [
+      {
+        "name": "name",
+        "label": "Title",
+        "placeholder": "e.g. Shapes of Desert Plants",
+        "help": "Enter a concise title.",
+        "required": true
+      },
+      {
+        "name": "description",
+        "label": "Description",
+        "help": "Add a thorough description with as much detail as possible. You can use [Markdown](https://daringfireball.net/projects/markdown/basics).",
+        "type": "textarea",
+        "required": true
+      },
+      {
+        "name": "files",
+        "label": "File",
+        "placeholder": "e.g. https://file.com/file.json",
+        "help": "Please provide a URL to your algorith file. This URL will be stored encrypted after publishing.",
+        "type": "files",
+        "required": true
+      },
+      {
+        "name": "dockerImage",
+        "label": "Docker Image",
+        "placeholder": "e.g. python3.7",
+        "help": "Please select a predefined image to run your algorithm.",
+        "type": "select",
+        "options": ["NodeJS", "Python 3.7"],
```

Comment:
since it's Docker images might be more clear if we reference them in the UI by actual image name in that Docker image syntax, like `node:latest` & `python:latest` or whatever specific images we use

Reply:
shouldn't we use options like `node:pre-defined` & `python:pre-defined`, or are we going to use the `latest` version of those two every time as pre-defined images?

Reply:
right, totally, we should use specific versions. Probably whatever is their latest LTS release or however that is called in the Python world